README.md

```diff
@@ -82,6 +82,9 @@ This snippet, taken from [example dir](https://github.com/appium-userland/appium
 | appium:maxRetryCount    | The count for socket connection retry for get flutter session (default 30)          | 20|
 | appium:observatoryWsUri | The URL to attach to the Dart VM. The appium flutter driver finds the WebSocket URL from the device log by default. You can skip the finding the URL process by specifying this capability. Then, this driver attempt to establish a WebSocket connection against the given WebSocket URL. Note that this capability expects the URL is ready for access by outside an appium session. This flutter driver does not do port-forwarding with this capability. You may need to coordinate the port-forwarding as well. | 'ws://127.0.0.1:60992/aaaaaaaaaaa=/ws' |
 | appium:skipPortForward | Whether skip port forwarding from the flutter driver local to the device under test with `observatoryWsUri` capability. It helps you to manage the application under test, the observatory URL and the port forwarding configuration. The default is `true`. | true, false |
+| appium:remoteAdbHost | The IP/hostname of the remote host ADB is running on. This capability only makes sense for Android platform. Providing it will implicitly override the host for the Observatory URL if the latter is determined from device logs. localhost be default | 192.168.1.20
+| appium:adbPort | The port number ADB server is running on. This capability only makes sense for Android platform. 5037 by default | 9999
+| appium:forwardingPort | The port number that will be used to forward the traffic from the device under test to locahost. Only applicable if `skipPortForward` is falsy. Not applicable if the test is executed on iOS Simulator. By default, it is the same as in the provided or autodetected Observatory URL. | 9999
```

Comment:
This driver can work with a real device as well, so here can just mention iOS.

```suggestion
| appium:forwardingPort | The port number that will be used to forward the traffic from the device under test to locahost. Only applicable if `skipPortForward` is falsy. Not applicable if the test is executed on iOS. By default, it is the same as in the provided or autodetected Observatory URL. | 9999
```

Reply:
Unfortunately the port forwarding only exists for real devices. iOS simulator opens the port on the same host where it is running, so no forwarding is needed there. Check https://github.com/appium-userland/appium-flutter-driver/blob/849b9f532c69cc6d9157642b54e3130aa8cc2708/driver/lib/sessions/ios.ts#L105

Reply:
ah, it is. I've added it when added supporting a real device. (forgot as I mostly used real devices)